sunpy/net/jsoc/jsoc.py

```diff
@@ -420,6 +420,13 @@ def request_data(self, jsoc_response, **kwargs):
         jsoc_response : `~sunpy.net.jsoc.jsoc.JSOCResponse` object
             The results of a query
 
+        method : `str`
```

Comment:
```suggestion
        method : {'url-tar', 'url', 'url-quick'}
```

Reply:
I dont understand this change, i assumed that we want to say what data type given parameter is.

Reply:
We try to follow the https://numpydoc.readthedocs.io/en/latest/format.html guide for our docs.
For parameters that have a fixed set of values, this is the format.